docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,465 @@ import { StyledContent } from '../../components/styled.elements';
-The segment after `app.redocly.com/org/` is your organization ID.
-For example, if the URL is `app.redocly.com/org/test_docs`, the organization ID is `test_docs`.
+When you run the [preview-docs command](../commands/preview-docs.md), you *must* specify a file. As a shortcut, you can reference definitions listed in the `apis` section.
+
+```shell
+openapi preview-docs production
+```
+
+### Optional properties supported by the `apis` section
+The `apis` section supports several optional properties for each definition file listed.
 
+#### The `labels` property
+Use `labels` to assign one or more existing Redocly Workflows organization-wide labels to your OpenAPI definitions. You must have a Redocly Workflows account with an active organization, and the top-level `organization` property must be added to `redocly.yaml`.
 
-### apis
+```yaml
+apis:
+  main:
+    root: ./openapi/openapi.yaml
+    labels:
+      - main
+```
 
-The `apis` section is used to configure one or more APIs. Every API in the section is identified by its name and version in the format `name@version`. The version is optional, and when not provided, Redocly apps interpret it as `latest` by default. Every `name@version` combination listed in the section must be unique.
+:::info
+If you try to assign labels that don't already exist for your organization, Redocly apps will show a warning in the output and will only assign existing labels.
+:::
 
-For every API listed in the section, you must provide the path to the OpenAPI definition using the `root` property.
+#### The `lint` property
+In addition to the global (top-level) `lint` section, you can define lint rules for each definition listed in the `apis` section.
 
 ```yaml
 apis:
-  name@version:
+  main:
     root: ./openapi/openapi.yaml
     labels:
-      - production
-    lint: []
-    features.openapi: []
+      - main
+    lint:
+      rules:
+        example-rule: warn
+```
+
+Not all `lint` options are supported for individual APIs.
+
+Option | Per-API support | Behavior
+-- | -- | --
+lint.extends | ❗ | API settings override global settings.
+lint.rules | ✅ | API and global settings merge. In case of conflict, API takes priority.
+lint.decorators | ✅ | API and global settings merge. In case of conflict, API takes priority.
+lint.preprocessors | ✅ | API and global settings merge. In case of conflict, API takes priority.
+lint.plugins | ❌ | Available in global configuration only.
+
+#### The `features.openapi` property
+The global (top-level) `features.openapi` section is where you configure how all of your published reference docs will look (style, fonts, colors — known as *theming*) and behave (enable/disable search, show/hide API reference docs download option — known as *functionality*).
+
+If you need to apply different theming and functionality to individual APIs, just add the `features-openapi` property to the `apis` section, and use the same options as the global `features-openapi`.
```

Comment:
```suggestion
If you need to apply different theming and functionality to individual APIs, add the `features-openapi` property to the `apis` section, and use the same options as the global `features-openapi`.
```

Reply:
issue: I don't understand this second part:

> and use the same options as the global features-openapi.

Do you mean to say that users should copy the contents of the global `features.openapi` into each individual API's `features.openapi`? Or that the same options supported in the global section are supported in individual sections? Like this sentence from the original text? "This section supports identical format and options as the global features.openapi section."

Also, it should be `features.openapi` - fullstop, not hyphen.